src/phytoplankton.F90

```diff
@@ -204,6 +204,8 @@ subroutine initialize(self, configunit)
       call self%register_state_dependency(self%id_tal, standard_variables%alkalinity_expressed_as_mole_equivalent)
       call self%register_state_dependency(self%id_oxy, 'oxy', 'mol O2 L-1', 'oxygen')
 
+      call self%register_state_dependency(self%id_fer, 'fer', 'mol Fe L-1', 'iron')
```

Comment:
since `fer` is only used for calcification, could this be made conditional on `self%calcify`? In that case the  `_GET_(self%id_fer,  fer)` should also moved down into the `if ( self%calcify)` section, and the `fer` coupling for diatoms can be dropped from `fabm.yaml`.